Drop-In/src/main/java/com/braintreepayments/api/dropin/AddCardActivity.java

```diff
@@ -324,12 +334,15 @@ public void onSmsCodeSent(String enrollmentId, boolean smsRequired) {
     @Override
     public void onCancel(int requestCode) {
         if (requestCode == BraintreeRequestCodes.THREE_D_SECURE) {
+            mRequestedThreeDSecure = false;
```

Comment:
^ I second Susan's input on the name of this variable being changed. This at first reads as that we are setting the merchant's request of 3DS to false.